guides/common/modules/con_provisioning-virtual-machines-on-kubevirt.adoc

```diff
@@ -5,7 +5,7 @@
 This technology provides a unified development platform where developers can build, modify, and deploy applications residing in application containers and VMs in a shared environment.
 These capabilities support rapid application modernization across the open hybrid cloud.
 
-With {Project}, you can create a compute resource for {KubeVirt} so that you can provision and manage virtual machines in {Kubernetes} using {Project}.
+With {Project}, you can create a compute resource for {KubeVirt} so that you can provision and manage virtual machines in {Kubernetes} by using {Project}.
```

Comment:
```suggestion
You can create a compute resource for {KubeVirt} so that you can provision and manage virtual machines in {Kubernetes} by using {Project}.
```